libs/media-player/src/lib/shared/canvas/canvas.tsx

```diff
@@ -0,0 +1,26 @@
+import { SxProps, styled } from '@mui/material';
+import { Draw, useCanvas } from './use-canvas';
+
+const StyledCanvas = styled('canvas')({
+    display: 'block',
+    margin: 0,
+    padding: 0,
+});
+
+interface CanvasProps {
+    canvasId: string;
+    draw: Draw;
+    width: number;
+    height: number;
+    sxProps: SxProps;
+}
+
+export const Canvas = ({ canvasId, draw, width, height, sxProps }: CanvasProps) => {
```

Comment:
We might need to break this out somewhere other than the media player lib in the future. But for now, this is good.